### PR TITLE
Gerrit: Added location docs

### PR DESCRIPTION
**Pull request by anicke**
Documentation have been added that describes how to configure Gerrit
integrations.

Relates: #9896

Signed-off-by: Niklas Aronsson <niklasar@axis.com>

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [X] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by pjungermann — APPROVED** [review]

**Review by jhaals — APPROVED** [review]
Thanks for the documentation @anicke!

File: docs/integrations/gerrit/locations.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Configuration
+
+To use this integration, add at least one Gerrit configuration to your root `app-config.yaml`:
+
+```yaml
+integrations:
+  gerrit:
+    - host: gerrit.company.com
+      apiBaseUrl: gerrit.company.com/gerrit
+      gitilesBaseUrl: gerrit.company.com/gitiles
```
Comment (freben) [inline docs/integrations/gerrit/locations.md:23]: base URLs typically contain a protocol (making them proper URLs as the name implies) - shouldn't these?

Comment (freben) [reply docs/integrations/gerrit/locations.md:23]: Yeah I see in the examples below that that's what seems to be expected

**Comment by freben** [comment]
One more thing - to get this to appear on the microsite, you need to add it in the appropriate place in `microsite/sidebars.json` too.

File: docs/integrations/gerrit/locations.md (hunk 1 above)
Comment (anicke) [reply docs/integrations/gerrit/locations.md:23]: Yes, you are correct! I'll fix that and the "microsite" asap!

Comment (anicke) [reply docs/integrations/gerrit/locations.md:23]: Fixed in #10721 10721

Comment (anicke) [reply docs/integrations/gerrit/locations.md:23]: Fixed in PR #10721

**Pull request merged**